common/src/test/java/com/hartwig/serve/common/classification/matchers/PresenceAbsenceOfProteinMatcherTest.java

```diff
@@ -0,0 +1,43 @@
+package com.hartwig.serve.common.classification.matchers;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Set;
+
+import com.google.common.collect.Sets;
+
+import org.junit.Test;
+
+public class PresenceAbsenceOfProteinMatcherTest {
+
+    private static final Set<String> PRESENCE_OF_PROTEIN_EVENTS = Sets.newHashSet("positive");
+    private static final Set<String> PRESENCE_OF_PROTEIN_BLACKLIST_EVENTS = Sets.newHashSet("HRD", "MSI");
+    private static final Set<String> ABSENCE_OF_PROTEIN_EVENTS = Sets.newHashSet("negative");
+    private static final Set<String> ABSENCE_OF_PROTEIN_BLACKLIST_EVENTS = Sets.newHashSet("HRD", "MSI");
+
```

Comment:
Minor and maybe just my preference, but I typically never test with "real" values. I would for instance put "blacklist" in the blacklist events rather than "real" values. In the end you want it to work for the abstract case, of which the concrete implementation is an example.

Reply:
Can you check if you does mean this? (see my changes)